### DISCUSSION TITLE
와 미친

### DISCUSSION THREAD

**Discussion by lhwdev** [lhwdev/covid-selftest-macro#4] [근황]
자가진단 앱의 내부 구조가 바뀌어 자가진단(뿐만 아니라 그냥 로그인 자체)가 안되는 걸 확인했습니다.
그 원흉은 'validatePassword'라는 비밀번호를 확인하는 것에 있는데요, 자가진단 앱에서 비밀번호 입력란을 보면 보안 키보드가 적용되어 있습니다.
그에 맞게 내부 구조도 바뀌었고요.
하... 이번에는 좀 복잡하게 바뀌어서 구조 분석하는데 시간이 좀 걸리겠네요.
조금만 기다려주세요.

원래 하고있던 3.0 업데이트는 아직 완성되기까지 한참 남았으니 2.9 버전에서 가져와서 할 수 밖에 없네요.
ㅠㅠ

**Comment by junseok0304** [comment]
안녕하세요 바뀐 보안키패드 구조 관련해서 도움 드리고 싶습니다.
개인 이메일 주소와 함께
답장 남겨주세요 감사합니다.

2021년 7월 24일 (토) 오전 9:54, LHW ***@***.***>님이 작성:

**Reply by lhwdev** [reply]
README에도 있다시피 제 개인 이메일 주소는 lhwdev6@outlook.com 입니다.
도움을 주신다면 정말 감사하겠습니다.
(지금 코드 뜯어보고 있는데 쉽지는 않네요. 그냥 그 js 라이브러리(transkey)를 째로 실어야 하는지..)
